openmdao/test_suite/test_examples/test_circuit_analysis.py

```diff
@@ -0,0 +1,236 @@
+from __future__ import print_function, division, absolute_import
+import numpy as np
+
+import unittest
+
+from openmdao.api import ExplicitComponent, ImplicitComponent, Group, NewtonSolver, DirectSolver
+
+from openmdao.devtools.testutil import assert_rel_error
+
+
+class Resistor(ExplicitComponent):
+    def initialize(self):
+        self.metadata.declare('R', default=1., desc='Resistance in Ohms')
+
+    def setup(self):
+        self.add_input('V_in', units='V')
+        self.add_input('V_out', units='V')
+        self.add_output('I', units='A')
+
+        self.declare_partials('I', 'V_in', method='fd')
+        self.declare_partials('I', 'V_out', method='fd')
+
+    def compute(self, i, o):
```

Comment:
Not so sure about shorthanding "inputs" and "outputs" to i and o.  Seems best for learners to see the full names. Also, i and o aren't pep-8 compliant, and of all the letters, i and o are the most confusable (i for 1 or imaginary numbers, and o for zero).